packages/next/src/build/collect-build-traces.ts

```diff
@@ -395,6 +395,14 @@ export async function collectBuildTraces({
         const { entryNameFilesMap } = buildTraceContext?.chunksTrace || {}
 
         const cachedLookupIgnoreRoutes = new Map<string, boolean>()
+        let chunks: string[] = []
+
+        // we aren't getting all chunks in the trace-entrypoint plugin
+        // with rspack currently so for now just add them manually for
+        // all trace files
+        if (process.env.NEXT_RSPACK) {
```

Comment:
Since we're checking this in multiple places in this file, let's hoist this into a constant at the top?